README.md

```diff
@@ -9,31 +9,42 @@ JWST Calibration Pipeline
 
 Note
 ----
-Beginning with the next version (0.9.0), **JWST requires Python 3.5 and above**.
+Beginning with version 0.9.0, **JWST requires Python 3.5 and above**.
 
 Installing
 ----------
-Releases are made on the **stable** branch. To install the latest release:
 
-    % git clone https://github.com/STScI-JWST/jwst.git
+To install the latest released version of the library, we recommend using [conda](https://conda.io/docs/index.html)
+and the [astroconda](https://astroconda.readthedocs.io) channel to install the
+software and using a conda environment for the installation:
 
-    % cd jwst
+    conda create -n jwst --override-channels -c http://ssb.stsci.edu/astroconda-dev -c defaults jwst
+    source activate jwst
 
-    % git checkout stable
+This will install the latest released version of the package, and within the
+`jwst` environment, the pipeline can be run from the command line or imported
+into python programs.   In order to install a specific version, the exact 
+version can be supplied, e.g. jwst=0.9.0. 
 
-    % python setup.py install
+To install the development version of the repository, we recommend creating a new
+environment, using the [astroconda](https://astroconda.readthedocs.io) channel
+to install the dependencies, and then installing from the github repository:
 
-The main development is on the **master** branch. To install the development version:
+    conda create -n jwst_devel --only-deps --override-channels -c http://ssb.stsci.edu/astroconda-dev -c defaults jwst
+    source activate jwst
+    git clone https://github.com/STScI-JWST/jwst.git
```

Comment:
If the user is going to contribute code, they should fork the repo and then clone their fork locally.  But maybe we don't want to get into that here.  Keeping it simple here is good.